File: easy/1 TwoSum/rio/twoSum.c
#include <stdlib.h>
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    int* result = malloc(2 * sizeof(int));
     *returnSize = 0 ;
    for(int i = 0; i < numsSize; i++){
        int remain = target - nums[i];
        for(int k  = i + 1; k < numsSize; k++){
            if(nums[k] == remain){
              result[0] = i;
              result[1] = k;
              *returnSize = 2;
              return result;
            }
        }
    }
    return NULL;
}

File: easy/1 TwoSum/rio/twoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var res = [0,0];
    for(let i = 0; i < nums.length -1; i++){
        let remainVal = target - nums[i];
        for( let j = i+1; j < nums.length; j++){
            if(nums[j] == remainVal){
                res[0] = i;
                res[1] = j;
                break;
            }
        }
    }
    return res;
};